lib/index.js

```diff
@@ -97,6 +97,31 @@ PostHTML.prototype.process = function (tree, options) {
   tree = options.skipParse ? tree : parser(tree, options)
 
   tree.options = options
+  /**
+  * Messages to store and pass metadata
+  *
+  * @memberof tree
+  *
+  * @type {Array} messages
+  *
+  * @example
+  * ```js
+  * export default function plugin (options = {}) {
```

Comment:
support node v0.10

Reply:
It's just JSDoc example 🙃 :) ? But I can use CJS (which might be better anyways)

Reply:
done